core/src/main/java/com/ibm/wala/ipa/cha/ClassHierarchy.java

```diff
@@ -706,6 +708,97 @@ private void recursiveStringify(Node n, StringBuilder buffer) {
     }
   }
 
+  // Puts ClassHierarchy into a JSON readable variable that maps the class to a list of its
+  // subclasses
+  public Object toJson() {
+    // lines 713-715: initialize variables (subclass and dag) to store important values like hashmap
+    // of each iteration and a list of all maps of class to subclasses
+    HashMap<Object, Object> subclass = new HashMap<>();
```

Comment:
Can this be declared as `Map<String,Set<String>>` instead?  That will make it a lot clearer what is going on; that this data structure maps a string representation of each class / interface to all of its immediate subtypes